src/TTextEdit.cpp

```diff
@@ -54,6 +54,7 @@
 #include <QTextBoundaryFinder>
 #include <QToolTip>
 #include <QVersionNumber>
+#include <QStringRef>
```

Comment:
IIRC We try to keep these in alphabetical order, so it should go up a few lines...